services/worker-manager/schemas/constants.yml

```diff
@@ -118,7 +118,7 @@ scope:
 source-pattern: "^(https?:\/\/|ssh:\/\/|git@)"
 
 # Maximum number of dependencies a single task can have
-max-task-dependencies: 100
+max-task-dependencies: 10000
```

Comment:
Do you know why this was also defined in worker manager? Shouldn't it be a Queue-only thing? And shouldn't it share the same config value if it is really needed, rather than having a duplicate one that can become out-of-sync?

Reply:
great question! 
so this is coming from this feature https://github.com/taskcluster/taskcluster/pull/5448 which merged some views (queue+w-m)
but, now as I'm checking this values is not really  needed on this side, so I'd drop it. `task.yml` is being defined in schemas only to reference `workerPoolId`, `workerType` and `provisionerId` which we can describe without moving `task.yml`

Reply:
I create a separate task for this - https://github.com/taskcluster/taskcluster/issues/7242